api/services/ocean/aks/responses/rightSizingResponse.yaml

```diff
@@ -0,0 +1,63 @@
+description: Ocean RightSizing Recommendation Response
```

Comment:
The file name is not by the convension need to be like rightSizingCreate

Reply:
Modified to rightSizingSearch